generators/Exercises/TwoBucket.cs

```diff
@@ -0,0 +1,49 @@
+using Generators.Input;
+using Generators.Output;
+
+namespace Generators.Exercises
+{
+    public class TwoBucket : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in canonicalData.Cases)
+            {
+                canonicalDataCase.TestedMethodType = TestedMethodType.Instance;
+                canonicalDataCase.SetConstructorInputParameters(new [] { "bucket_one", "bucket_two", "start_bucket"});
```

Comment:
The `SetConstructorInputParameters` take a `param string[]`, which means you can simplify this to: `SetConstructorInputParameters("bucket_one", "bucket_two", "start_bucket")`